src/submodules/helper.ts

```diff
@@ -1,12 +1,12 @@
 import { ethers, Contract } from 'ethers'
-import JSONBig from 'json-bigint'
+import * as JSONBig from 'json-bigint'
```

Comment:
Why `import * as JSONBig from 'json-bigint'` this  and  not `import JSONBig from 'json-bigint'`